src/demo-app/a11y/list/list-a11y.html

```diff
@@ -0,0 +1,66 @@
+<div class="demo-list">
+  <section>
+    <h2> Seasoning </h2>
+    <p>Showing a non-interactive list of seasonings.</p>
+    <md-list role="list">
+      <md-list-item *ngFor="let item of items"> {{item}} </md-list-item>
```

Comment:
Shouldn't all of these list items have a `role="listitem"`?

Reply:
`role="listitem"`  is added in `list.ts` so we don't need to add it here.

Reply:
We're we going to remove that from the list-item?